test/rules/apex/apex-import.spec.ts

```diff
@@ -9,9 +9,10 @@ import { RuleTester } from '@typescript-eslint/rule-tester';
 
 import { rule, APEX_IMPORT_RULE_ID } from '../../../src/rules/apex/apex-import';
 import { createScopedModuleRuleName } from '../../../src/util/rule-helpers';
+import path from 'path';
 
 const ruleTester = new RuleTester({
-    parser: '@typescript-eslint/parser'
+    parser: path.resolve(__dirname + '../../../../node_modules/@typescript-eslint/parser/dist')
```

Comment:
Why are we changing this—or even providing a `parser` option here, since the [default](https://github.com/typescript-eslint/typescript-eslint/blob/d6e0f6e7da7b81d1e4b70d3342651c0e7e29c000/packages/rule-tester/src/RuleTester.ts#L68) is `@typescript-eslint/parser`? This is definitely a non-standard way to construct `RuleTester`. When I comment out the configuration locally, I don't get any errors using the default values?

Reply:
Forget to remove this test code.  I put it there to see if it would magically fix the module failure, but apparently not.